packages/redux/src/promotionEvaluations/actions/factories/fetchPromotionEvaluationItemsFactory.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  FETCH_PROMOTION_EVALUATION_ITEMS_FAILURE,
+  FETCH_PROMOTION_EVALUATION_ITEMS_REQUEST,
+  FETCH_PROMOTION_EVALUATION_ITEMS_SUCCESS,
+} from '../../actionTypes';
+import type { Dispatch } from 'redux';
+import type { FetchPromotionEvaluationItemsAction } from '../../types';
+import type {
+  GetPromotionEvaluationItems,
+  PromotionEvaluationId,
+  PromotionEvaluationItem,
+} from '@farfetch/blackout-client/promotionEvaluations/types';
+
+/**
+ * @callback FetchPromotionEvaluationItemsThunkFactory
+ *
+ * @param {number} promotionEvaluationId - Numeric identifier of the promotion.
```

Comment:
Hmm, please review this, in the client this is a string and has a different description 🧐

Reply:
It should always be a number. I fixed it everywhere I believe.